alerts/report_sources/alerts_mambasa_2019-12-10.Rmd

```diff
@@ -1,2810 +1,2856 @@
-  large_txt +
-  rotate_x_text(45) +
-  theme(legend.position = "bottom") +
-  scale_x_date(date_breaks = "1 week", date_labels =  "%d %b") 
-  
-```
-
-
-### Table - daily past 3 weeks
-
-```{r table_ha_over_time_recent_decisions}
-
-table_ha_over_time_recent_decisions <- outcomes_recent %>%
-  count(date, top_aires, decision_comparison) %>%
-  spread(decision_comparison, n, fill = 0) %>%
-  adorn_totals(where = c("row", "col"))
-
-table_ha_over_time_recent_decisions %>%
- show_table()
-  
-```
-
-
-<!-- ======================================================= -->
-<!-- ======================================================= -->
-<!-- ======================================================= -->
-
-# Alert Rates {.tabset .tabset-fade .tabset-pills}
-
-## Outline
-
-This section integrates information on the number of alerts with population data to estimate the number of validated/invalidated alerts per 10,000 people.
-
-**Note:** Rate are only calculated for areas with population data. There 
-are `r length(non_pop_data)` Aire de Santes without population data, which totals
- **`r alerts_no_pop`** alerts.
-
-**Note: All known Aire de Santes are reported in tables**
-
-
-
-```{r reactivate_alert_rates, include = FALSE}
-
-if(allNA(x$as_population)){
-  knitr::opts_chunk$set(eval = FALSE)
-  no_pop_data <- TRUE
-  
-} else{
-  cat("There is no population data therefore alert rates can not be calculated.")
-}
-
-```
-
-
-<!-- ======================================================= -->
-## Overall
-
-### Weekly, Since Database Start 
-
-```{r alert_rates}
-
-x_alert_rate_sous_co <- x %>%
-  count(epiweek_report_label, status, total_population) %>%
-  mutate(alert_rate = 10000 * n / total_population) 
-
-ggplot(data = x_alert_rate_sous_co, 
-       aes(x = epiweek_report_label, y = alert_rate, fill = status)) +
-  geom_col() + 
-  scale_validations + 
-  labs(x = "",
-       y = "Nombre d'alertes par semaine \net par 10,000 personnes",
-       title = paste0("Nombre d'alertes par 10000 personnes - Mambasa")) +
-  rotate_x_text(45) +
-  theme(legend.position = "bottom") +
-  scale_months
-
-```
-
-
-### Table - Weekly Since Database Start
-
-```{r table_alert_rates_per_week, fig.keep = "all"}
-
-table_alert_rates_per_week <- x_alert_rate_sous_co %>%
-  select(-alert_rate) %>% 
-  spread(status, n, fill = 0) %>% 
-  mutate(total = statut_inconnu + invalidee + validee,
-         alert_rate = 10000 * total / total_population,
-         alert_rate = round(alert_rate, 4))
-
-table_alert_rates_per_week %>%
-  show_table()
-
-```
-
-
-<!-- ======================================================= -->
-## Overall Past 3 Weeks
-
-### Daily, Past 3 Weeks 
-
-``` {r alert_rates_recent}
-
-x_alert_rate_sous_co <- x_recent %>%
-  count(date, status, total_population) %>%
-  mutate(alert_rate = 10000 * n / total_population) 
-
-ggplot(data = x_alert_rate_sous_co, aes(x = date, 
-                                        y = alert_rate, fill = status
```

Comment:
I'm not that familiar with the alerts data, but is it worth including these as unknown?